<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS><TS version="2.0">
<context>
    <name>PluginErrorDialog</name>
    <message>
        <location filename="traceback_widget.py" line="41"/>
        <source>Plugin error report</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="traceback_widget.py" line="44"/>
        <source>Some plugins have errors and were removed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="traceback_widget.py" line="49"/>
        <source>Accept</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>TracebackWidget</name>
    <message>
        <location filename="traceback_widget.py" line="70"/>
        <source>Traceback</source>
        <translation type="unfinished"></translation>
    </message>
</context>
</TS>
